Signal/src/ViewControllers/MessagesViewController.m

```diff
@@ -3210,6 +3226,8 @@ - (void)endRecordingVoiceMemo
         DDLogInfo(@"Discarding voice message; too short.");
         self.audioRecorder = nil;
 
+        [self dismissKeyBoard];
```

Comment:
This alert can overlap the keyboard on small form factors.  I'm not sure if we need to solve problem in other places as well.